cypress/e2e/shared/nav.test.ts

```diff
@@ -157,11 +162,23 @@ describe('navigation', () => {
       'variables',
       'templates',
       'labels',
-    ].forEach(x => {
-      cy.getByTestID(`nav-subitem-${x}`)
+    ].forEach(navItem => {
+      if (navItem === 'history') {
+        cy.getByTestID('nav-item-alerting').click()
+      } else if (
+        navItem === 'variables' ||
+        navItem === 'templates' ||
+        navItem === 'labels'
+      ) {
+        cy.getByTestID('nav-item-settings').click()
+      } else {
+        cy.getByTestID('nav-item-load-data').click()
+      }
```

Comment:
why is this needed now?

Reply:
In the current Nav, all sub menu are expanded by default. With the new nav, submenus are hidden by default, visible only when expanded. So this logic takes care of expanding the submenu

Reply:
In our previous UI, sub menu items were open and visible by default like this: 

<img width="250" alt="Screen Shot 2021-09-03 at 11 24 55 AM" src="https://user-images.githubusercontent.com/18511823/132051003-8efc654f-0c86-45b6-ab2a-b6c9b4369cd3.png">


new UI, we have to click on them to open the sub menu items. So, this adds a little check.

<img width="259" alt="Screen Shot 2021-09-03 at 11 28 14 AM" src="https://user-images.githubusercontent.com/18511823/132051019-a3cb7822-08a0-4005-8ec6-73d60c7bb05c.png">